src/hardware/targets.json

```diff
@@ -224,15 +224,15 @@
                 "upload_methods": ["uart", "wifi", "betaflight"]
             },
             "diversity": {
-                "product_name": "Generic ESP32 2.4Ghz Diversity RX",
+                "product_name": "Generic ESP32 Full Diversity PA 2.4Ghz RX",
```

Comment:
Minor, but we've been using "True Diversity" to differentiate these receivers from Antenna diversity.

What do we think about changing the 4 instances below from Full Diversity to True Diversity?

Reply:
_100_